internal/cephfs/clone.go

```diff
@@ -43,6 +43,27 @@ const (
 	snapshotIsProtected = "yes"
 )
 
+// toError checks the state of the clone if it's not cephFSCloneComplete.
+func (cs cephFSCloneState) toError(ctx context.Context, cloneID volumeID) error {
+
```

Comment:
nit: empty line not needed.

I wonder if the exhaustive linter notices that `cephFSCloneComplete` is not in the switch.

Reply:
okay yes. I will rerun the exhaustive linter locally and update the PR if needed.